Add note that the service is charged for (#3262)
elastic/observability-docs: docs/en/observability/synthetics-reference/global-managed-paid-for.asciidoc

> Please note that Elastic’s global managed testing infrastructure is a charged for service, as shown on the https://www.elastic.co/pricing[pricing page].

Reviewer: The phrase "is a charged for service" sounds strange to me. What about something like this? Not sure if it should be "when you run monitors" or "when you create monitors":

```suggestion
When you run monitors on Elastic’s global managed testing infrastructure, you will be charged as outlined on the https://www.elastic.co/pricing[pricing page].
```

Author: Thanks @colleenmcginnis that's much better than what I had written as I wasn't sure what @drewpost wanted to include.

@drewpost are you happy to go with this, or something else?

Reviewer 2: How about. "Executing synthetic tests on the Elastic global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the type of test. Browser tests are charged for on a per test run basis and lightweight ping tests have an all you can eat model charged by region. Please refer to https://www.elastic.co/pricing for full details and current pricing" @paulb-elastic @colleenmcginnis

> Executing synthetic tests on the Elastic global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the type of test. Browser tests are charged for on a per test run basis and lightweight ping tests have an all you can eat model charged by region. Please see https://www.elastic.co/pricing[full details and current pricing].

Reviewer: A couple suggestions:

* Throughout the docs we tend to use "Elastic's global managed testing infrastructure" instead of "the Elastic global managed testing infrastructure".
* Throughout the docs we use "monitor types" instead of "type of test".
* The phrase "all you can eat model" might not translate well. I attempted a generic description that doesn't rely on familiarity with the idiom, but feel free to adjust.
* As a general rule, we avoid using "see" in favor of "refer to".

```suggestion
Executing synthetic tests on Elastic's global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the monitor type. _Browser monitor_ usage is charged for on a per test run basis. _Lightweight monitor_ usage is charged a flat fee by region regardless of the number of runs. For more details, refer to https://www.elastic.co/pricing[full details and current pricing].
```

Author: Thank you @colleenmcginnis, @drewpost, ok with this?

Reviewer 2: @colleenmcginnis I'm fine with the standards type changes.

In terms of the lightweight tests, I think it's slightly confusing as if I see flat-fee on one but not the other, it makes me think there's some variable on the browser tests that isn't a flat rate. Could we use a slightly different term? FWIW, this unlimited consumption model is unique in the industry so we'd want to really emphasise it.

Reviewer: >it makes me think there's some variable on the browser tests that isn't a flat rate

Isn't the number of test runs that variable? I guess maybe it depends on if there's a line item that says "Browser monitors" that is the sum of all runs or if each run is listed as a single line item? You know the pricing structure better than me so I trust your suggestions as long as we avoid idioms. 🙃  

Maybe as simple as "For _Lightweight monitor_ usage you're charged one fee per region regardless of the number of runs." ?

Reviewer 2: <img width="1360" alt="image" src="https://github.com/elastic/observability-docs/assets/7509768/a2a63b2c-90bd-4847-8a87-b0cfa909a37c">

This is what a customer would see in their cloud billing console. So for browser tests we charge a flat rate per test. So a customer running a test would be charged $0.0123 for up to 60 seconds of test execution. For lightweight tests, they simply are paying for capacity per region and can execute as many lightweight tests as they want within that capacity.

Reviewer: Ok. How about something like this? 

```suggestion
Executing synthetic tests on Elastic's global managed testing infrastructure incurs an additional charge. Tests are charged under one of two new billing dimensions depending on the monitor type. For _browser monitor_ usage, there is a fee per test run. For _lightweight monitor_ usage, there is a fee per region in which you run any monitors regardless of the number of test runs. For more details, refer to https://www.elastic.co/pricing[full details and current pricing].
```

Reviewer 2: Perfect @colleenmcginnis LGTM
